src/Combinatorics/Graphs/functions.jl

```diff
@@ -1339,3 +1339,20 @@ function laplacian_matrix(g::Graph)
   A = matrix(ZZ, adjacency_matrix(g))
   return D-A
 end
+
+@doc raw"""
+    is_bipartite(g::Graph)
+
+Returns true if the graph `g`  is bipartite.
+
+# Examples
+```
+julia> g = graph_from_edges([1,2],[2,3],[3,4]]);
+
+julia> is_bipartite(G)
+true
+```
+"""
+function is_bipartite(g::Graph{T}) where T
+  return Polymake.graph.Graph{Undirected}(ADJACENCY=g.pm_graph).BIPARTITE
```

Comment:
```suggestion
  return Polymake.graph.Graph{Undirected}(ADJACENCY=g.pm_graph).BIPARTITE::Bool
```
Iirc, julia cannot see the types of polymake fields. So we should help it like this